obc_transceiver_simulator/packets.py

```diff
@@ -84,7 +90,11 @@ def receive_rx_packet():
 # Use `bytearray` instead of `bytes`
 def send_raw_uart(uart_bytes):
     print("Sending UART (%d bytes):" % len(uart_bytes), bytes_to_string(uart_bytes))
-    Global.serial.write(uart_bytes)
+    # Drop packets?
+    if (random.uniform(0,1) < Global.uplink_drop):
+        print("Uplink Packet Dropped")
+    else:
+        Global.serial.write(uart_bytes)
```

Comment:
Move this logic to line 125 (in `send_tx_packet()`)

Reply:
Should I also put this in the 'Enter raw hex for UART:' command (f) as well?

Reply:
No, I intended for the raw UART functionality to just always output whatever you tell it.